src/Compilers/RealParserTests/RandomRealParserTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using static System.Console;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.CodeAnalysis;
+
+[TestClass]
+public class RandomRealParserTests
+{
+    [TestMethod]
+    public void TestRandomDoubleStrings()
+    {
+        var start = DateTime.UtcNow;
+        var nextReport = start + TimeSpan.FromSeconds(10.0);
+        var elapsed = (DateTime.Now - start);
```

Comment:
It looks like these would be used for timing. But I don't see them referenced in this method.
Are they dead? If so, delete them?
If not, please use UtcNow everywhere.